<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Atarea</class>
 <widget class="QDialog" name="Atarea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tareas</string>
  </property>
  <layout class="QFormLayout" name="formLayout_5">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 247, 249);</string>
     </property>
     <property name="title">
      <string>Agregar Tarea</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Nombre:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="inNombre"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Fecha:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDateEdit" name="inFecha">
          <property name="maximumDateTime">
           <datetime>
            <hour>23</hour>
            <minute>59</minute>
            <second>59</second>
            <year>2021</year>
            <month>12</month>
            <day>31</day>
           </datetime>
          </property>
          <property name="minimumDateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>2020</year>
            <month>7</month>
            <day>27</day>
           </datetime>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Tipo:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="inTipo">
          <item>
           <property name="text">
            <string>Universidad</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Personal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Familiar</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Otro</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Prioridad:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="inPrioridad">
          <item>
           <property name="text">
            <string>1. Alto</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2. Medio</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3. Bajo</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
